tests/helpers.cpp

```diff
@@ -0,0 +1,40 @@
+#include <string>
+#include <memory>
+#include <vector>
+#include <iostream>
+#include "helpers.h"
```

Comment:
maybe change to something like "utility"

Reply:
helpers still not changed

Reply:
Changed it to common now